src/components/features/errors/page-error.tsx

```diff
@@ -2,10 +2,7 @@ import { Theme, Container, Grid, Typography } from '@mui/material';
 import { makeStyles } from '@mui/styles';
 import { useTranslation } from 'next-i18next';
 import React from 'react';
-import { File } from 'react-kawaii';
-
 import { PageContainer } from '@src/components/templates/page-container';
-import colorfulTheme from '@src/theme';
```

Comment:
I guess the theming can stay? Or is it linked to the kawaii?

Reply:
Ah never mind, it was only applied on the file that was linked to the kawaii package :)

Reply:
The import wasn't being used